### ISSUE TITLE
Don't allow multiple expressions of the same time unit

### ISSUE THREAD

**Issue by giovannicuccu** [schyntax/schyntax#10]
The actual specifications allow multiple expressions of the same time fields, m(5) m(10) for example. Perhaps I'm missing something here, but it seems a bad idea to me because it can lead to some subtle ambiguities. 
The main issue is with negation expressions: 
1. `m(!10)` means every minute except the tenth, 
2. `m(5..20 !10)` means minutes from 5 to 20 with the exclusion of 10, 
3. but `m(5..20) m(!10)` has a different meaning (to me) since I see it equivalent to `m(!10)` and different from `m(5..20 !10)` . 

I like how schyntax makes easy for the developer to write schdule expressions and I see this ambiguity as a potential drawback

**Comment by bretcope** [comment]
I agree that there's probably no good reason to allow it. I'm actually less worried about the ambiguity, and more worried about it being a mistake. Adding a second expression of the same type likely means you missed the fact that that expression already exists in the schedule, and that could lead to unexpected results.

I don't remember what my original motivation for allowing it was, but I think it's worth changing.
